ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py

```diff
@@ -84,6 +84,7 @@ def _underlay_links(self) -> list:
                         "ipv6_enable": get(uplink, "ipv6_enable"),
                         "sflow": {"enable": self.shared_utils.fabric_sflow_downlinks},
                         "structured_config": get(uplink, "structured_config"),
+                        "child_mgmt_vlan": get(peer_facts, "inband_mgmt_vlan"),
```

Comment:
If we flip like I suggest under the schema, the logic could be on the child to add a key like `peer_inband_ztp_vlan: <int>` which could then be reversed here to `inband_ztp_vlan`.